.github/workflows/tests.yml

```diff
@@ -1,15 +1,18 @@
 name: CI
+
 on:
   push:
     branches:
       - master
   pull_request:
+
 env:
   HOMEBREW_DEVELOPER: 1
   HOMEBREW_NO_AUTO_UPDATE: 1
+
 jobs:
   syntax:
-    if: github.repository == 'Homebrew/brew'
+    if: github.repository == 'Homebrew/brew' && github.event_name != 'push'
```

Comment:
I think it'd still be good to have this run on pushes to `master`, too (and possibly all others except `tap-syntax`)

Reply:
Why?

Reply:
@reitermarkus It's good to see (and be emailed about) whether a given commit fails when it's merged into `master` (because we don't require rebasing/merging in `master` when merging). This has helped me catch several real bugs when merging.